### DISCUSSION TITLE
2024/08/11/7/

### DISCUSSION THREAD

**Discussion by giscus** [MengZe2l/mengze2l.github.io#4] [General]
# 2024/08/11/7/

前言在PHP中执行SQL查询后，有人就不会读取数据库里面的内容，这里就写上三种读取数据库的方法 fetch_assoc方法使用 fetch_assoc 方法获取关联数组。对于 SELECT 查询，可以使用 fetch_assoc 方法从结果集中获取下一行作为关联数组。每次调用 fetch_assoc 将返回结果集的下一行，直到没有更多行为止。 123456789// fetch_assoc方法$r

https://mengze2.cn/2024/08/11/7/

**Comment by MengZe2l** [comment]
## 测试评论系统

评论评论评论评论

```JavaScript
const widgets = computed(() => (route.meta.aside || []) as Array<keyof typeof widgetList>)
```

**Reply by MengZe2l** [reply]
回复测试
